packages/pwashortcut.rb

```
@@ -0,0 +1,169 @@
+    echo \"def installer_$2():\" >> $FLASK_APP
+    echo \"    return render_template('installer.html')\" >> $FLASK_APP
+    echo \"@app.route('/$2/<path:path>')\" >> $FLASK_APP
+    echo \"def $2_path(path):\" >> $FLASK_APP
+    echo \"    if path.endswith(('brew.png', 'manifest.json')):\" >> $FLASK_APP
+    echo \"        return send_from_directory('$2/templates', path)\" >> $FLASK_APP
+    echo \" \" >> $FLASK_APP
+    #######################
+    echo \"Shortcut for ${2^} deployed!\"
+    echo \"Wait for server start and go to localhost:5000/$2/ for installing shortcut.\"
+    sleep 2
+    flask run
+elif [[ $1 = '-s' ]]; then
+    flask run
+else
+    while getopts \"p:y:f:*\" arg; do
+        OPTARG=$(echo ${OPTARG} | sed 's/\=//')
+        RUN=NO
+        case ${arg} in
+           p)
+               echo $OPTARG
+               export FLASK_APP=$OPTARG
+               ;;
+           y)
+               echo $OPTARG
+               export PYTHONPATH=$OPTARG
+               ;;
+           f)  
+               RUN=YES
+               flask run $OPTARG
+               ;;
+           *)  echo \"Error: unknown option '$arg'\"
+               ;;
+        esac
+    done
+    if [[ $RUN != YES ]]; then
+      flask run
+    fi
+fi
+EOF"
+    #########################################
+  end
+
+  def self.install
+    FileUtils.mkdir "#{CREW_PREFIX}/lib/pwa/" unless File.exists? "#{CREW_PREFIX}/lib/pwa/"
+    FileUtils.mkdir "#{CREW_PREFIX}/lib/pwa/tools/" unless File.exists? "#{CREW_PREFIX}/lib/pwa/tools/"
+    FileUtils.mv "../brew_transparent.png", "#{CREW_PREFIX}/lib/pwa/tools/brew.png"
+    system "install -Dm755 pwashortcut #{CREW_PREFIX}/bin/pwashortcut"
+    system "cp * #{CREW_PREFIX}/lib/pwa/tools/"
+    FileUtils.mv "main.py.bak", "#{CREW_PREFIX}/lib/pwa/main.py"
+  end
+  def self.postinstall
+      puts 
+      puts "To complete the installation, execute the following:".lightblue
+      puts "echo 'nohup pwashortcut &> /dev/null' >> ~/.bashrc".libhtblue
```

Not a good idea to use `&> /dev/null` since it will hide errors.